src/app/(withAuth)/works/[workId]/edit/page.tsx

```diff
@@ -1,9 +1,11 @@
-import { notFound } from 'next/navigation'
+import { notFound, redirect } from 'next/navigation'
+import { getServerSession } from 'next-auth/next'
```

Comment:
`getServerSession(authOptions)`の代わりに,  `getSession`を他のファイルと同様に使うと実装方法が統一できて良いかもです！(もとのコードでも実装そのものは正しいです！)

```suggestion
import { getSession } from '@/lib/auth'
```